ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -29,6 +29,24 @@ def reciprocal_(x, name=None):
     return ivy.inplace_update(x, reciprocal(x))
 
 
+@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def remainder(x, y, name=None):
+    return ivy.remainder(x, y)
+
+
+@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def remainder_(x, y, name=None):
+    return ivy.inplace_update(x, remainder(x, y))
+
+
+@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+@to_ivy_arrays_and_back
+def round(x, name=None):
+    return ivy.round(x)
```

Comment:
We shouldn't be adding any additional functions in this PR apart from the one linked to the issue.